packages/backend/src/models/RolesModel.ts

```diff
@@ -110,13 +128,23 @@ export class RolesModel {
                 ),
             )
             .where(`${RolesTableName}.organization_uuid`, organizationUuid)
```

Comment:
What would you think of baking this into the API of `getSystemRoles(filterByRole?:  string)` where no filter returns all?

Reply:
not sure what you mean..